index.html

```diff
@@ -196,5 +196,47 @@ <h2>About me</h2>
             </ul>
         </div>
     </section>
+
+    <footer>
+        <div class="icons">
+        <p class="attract">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            Full name
+            Email address
+            Enter text here
+            Get in touch
+        </p>
+        <form action="https://formspree.io/f/xeqdogov" method="post">
+                <div>
+                    <input type="text" name="fullname" placeholder="Full name" maxlength="30" required class="formel">
+                </div>
+                <div>
+                    <input type="email" name="email" placeholder="Email address" required class="formel">
+                </div>
+                <div>
+                   
+                
+                <textarea  name="message" maxlength="500" class="textarea" placeholder="Enter your text" required>
+                </textarea>
+                </div>
+
+                <button type="submit">
+                      Get in touch
+                </button>
```

Comment:
### Your Button
![button](https://user-images.githubusercontent.com/93039359/186456017-dd73e356-6dc3-455f-b9fc-98e3126cab04.png)

### Figma Button
![figma-button](https://user-images.githubusercontent.com/93039359/186456067-a1b8c410-df31-41b8-acb2-d3952e15170e.png)

- [ ] Please make your button design as shown in the Figma template :blush: to make your design look professional :+1: Please change width, height, padding, font-size, font-weight etc :thinking: